console/console-init/ui/src/modules/iot-certificates/components/IoTCertificate/IoTCertificate.tsx

```diff
@@ -37,7 +37,10 @@ export const IoTCertificate: React.FunctionComponent<IIoTCertificateProps> = ({
   const [onEditMode, setOnEditMode] = useState<boolean>(false);
 
   return (
-    <PageSection className={css(style.no_bottom_padding)}>
+    <PageSection
```

Comment:
page section doesn't need any id

Reply:
I will let it be in this one but will keep in mind in other modules